search.py

```diff
@@ -5,6 +5,20 @@
 
 # Generic Functions
 
+def prepare_terms_dict():
+    """
+    return a dictionary of terms for searching through abbreviation
+    Arguments: N/A
+    Return: [dictionary]: key: term name (regular spelling); value: term abbreviation
+    """
+    f = open('database/terms_dictionary.txt')
+    # sheet = [line.rstrip('\t') for line in open('terms_dictionary.txt')]
+    dic = {}
+    for line in  f:
+        line = line.split('\t')
+        if len(line) == 5:
+            dic[line[0]] = line[2].strip('()')
+    return dic
```

Comment:
Can be turned into a comprehension:

```python
return {
    line[0]: line[2].strip('()')
    for line in line.split('\t')
    if len(line) == 5
}
```

Reply:
Will do. I used `for line in open('filename')`. Let me know if that is a bad idea.